app/views/devise/sessions/new.html.erb

```diff
@@ -47,6 +47,7 @@
           },
         },
       ) %>
+  <%= hidden_field_tag :platform_authenticator_available, value: 'unknown', id: 'platform_authenticator_available' %>
```

Comment:
I don't know that it matters much one way or the other, but if we used `auth_params` in `SessionsController`, we could use `SimpleForm`'s `f` helper here.

```suggestion
  <%= f.input :platform_authenticator_available, as: :hidden, id: 'platform_authenticator_available' %>
```